tests/tests.py

```diff
@@ -29,55 +29,105 @@ def create_event(title='test', outcome='Success', name='John Doe'):
-    """Tests to make sure that URLs get directed to the right views.
+class TestLogic(TestCase):
+    """Tests to make sure that all the logic performs as expected.
 
-    Verifies that the correct information is captured from the URLs,
-    that the URLs get directed correctly (or that the correct error
-    is displayed), and that the correct templates get used.
+    Verifies all the logic being used to create the app. For instance,
+    all the urls are tested to make sure they are accepted and that
+    they call the correct views and templates, and the model methods
+    are checked to make sure they perform as expected. Also, the
+    get_event_or_404 function defined in views.py is tested to ensure
+    expected behavior.
     """
 
-    def test_valid_urls(self):
-        """Check that valid URLs get directed to the correct views."""
+    def test_index_url(self):
+        """Check that the index url receives an HTTP 200."""
+        url = reverse('major-event-log:index')
+        self.assertEqual(self.client.get(url).status_code, 200)
+
+    def test_event_details_url(self):
+        """Check that the event_details url receives an HTTP 200."""
+        uuid = create_event().id
+        url = reverse('major-event-log:event_details', args=[uuid])
+        self.assertEqual(self.client.get(url).status_code, 200)
+
+    def test_event_atom_url(self):
+        """Check that the event_atom url receives an HTTP 200."""
+        uuid = create_event().id
+        url = reverse('major-event-log:event_atom', args=[uuid])
+        self.assertEqual(self.client.get(url).status_code, 200)
+
+    def test_event_premis_url(self):
+        """Check that the event_premis url receives an HTTP 200."""
         uuid = create_event().id
-        urls = (reverse('major-event-log:index'),
-                reverse('major-event-log:event_details', args=[uuid]),
-                reverse('major-event-log:event_atom', args=[uuid]),
-                reverse('major-event-log:event_premis', args=[uuid]),
-                reverse('major-event-log:feed'),
-                reverse('major-event-log:about'))
-        for url in urls:
-            self.assertEqual(self.client.get(url).status_code, 200)
-
-    def test_correct_view_called(self):
-        """Check that URLs are resolved to the correct views."""
+        url = reverse('major-event-log:event_premis', args=[uuid])
+        self.assertEqual(self.client.get(url).status_code, 200)
+
+    def test_feed_url(self):
+        """Check that the feed url receives an HTTP 200."""
+        url = reverse('major-event-log:feed')
+        self.assertEqual(self.client.get(url).status_code, 200)
+
+    def test_about_url(self):
+        """Check that the about url receives an HTTP 200."""
+        url = reverse('major-event-log:about')
+        self.assertEqual(self.client.get(url).status_code, 200)
+
+    def test_event_details_view_called(self):
+        """Check that the event_details page calls the correct view."""
```

Comment:
I think we need to replace "page" with a different word. 

Perhaps "Check that the event_details URL pattern calls the correct view."